tool/change_metrics.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:io';
+
+/// Calculates the amount of formatting changes in a given directory of code.
```

Comment:
Can this go out in a separate PR?

Reply:
I, uh, in fact, didn't intend to commit this yet.

I'll take it out of the PR. :)